HW_3.2.js

```diff
@@ -0,0 +1,14 @@
+function multiply(x) {
+    if (arguments.length === 0) {
```

Comment:
Пуста умова

Reply:
Тут нема потреби в перевірці, але якби була, то краще вже:

```suggestion
    if (!x) {
```